src/hooks/useDegradedState.test.ts

```
@@ -0,0 +1,185 @@
+import { renderHook, act } from '@testing-library/react';
```

ESLint

[error] Member 'act' of the import declaration should be sorted alphabetically.

(sort-imports)